test/tests.yml

```diff
@@ -30,760 +47,872 @@
+    - name: openstack_rhel
+    - name: openstack_rockylinux
+    - name: vsphere_centos
+    - name: vsphere_default
+    - name: vsphere_flatcar
 
-- scenario: calico_containerd
-  initVersion: v1.27.7
+- scenario: install_containerd
+  initVersion: v1.28.7
   infrastructures:
-    - name: azure_default
+    - name: aws_amzn
+    - name: aws_centos
+    - name: aws_default
+    - name: aws_flatcar
+    - name: aws_rhel
+    - name: aws_rockylinux
     - name: azure_centos
+    - name: azure_default
     - name: azure_flatcar
     - name: azure_rhel
     - name: azure_rockylinux
+    - name: digitalocean_centos
+    - name: digitalocean_default
+    - name: digitalocean_rockylinux
+    - name: equinixmetal_centos
+    - name: equinixmetal_default
+    - name: equinixmetal_flatcar
+    - name: equinixmetal_rockylinux
     - name: gce_default
+    - name: hetzner_centos
+    - name: hetzner_default
+    - name: hetzner_rockylinux
+    - name: openstack_centos
+    - name: openstack_default
+    - name: openstack_flatcar
+    - name: openstack_rhel
+    - name: openstack_rockylinux
+    - name: vsphere_centos
+    - name: vsphere_default
+    - name: vsphere_flatcar
 
-- scenario: calico_containerd_external
-  initVersion: v1.27.7
+- scenario: install_containerd
+  initVersion: v1.29.2
   infrastructures:
     - name: aws_amzn
     - name: aws_centos
     - name: aws_default
+      runIfChanged: "(.prow/|addons/|examples/|hack/|pkg/|test/)"
     - name: aws_flatcar
     - name: aws_rhel
     - name: aws_rockylinux
-    - name: azure_default
     - name: azure_centos
+    - name: azure_default
     - name: azure_flatcar
     - name: azure_rhel
     - name: azure_rockylinux
-    - name: openstack_default
+    - name: digitalocean_centos
+    - name: digitalocean_default
+    - name: digitalocean_rockylinux
+    - name: equinixmetal_centos
+    - name: equinixmetal_default
+    - name: equinixmetal_flatcar
+    - name: equinixmetal_rockylinux
+    - name: gce_default
+    - name: hetzner_centos
+    - name: hetzner_default
+    - name: hetzner_rockylinux
     - name: openstack_centos
-    - name: openstack_rockylinux
-    - name: openstack_rhel
+    - name: openstack_default
     - name: openstack_flatcar
-    - name: vsphere_default
+    - name: openstack_rhel
+    - name: openstack_rockylinux
     - name: vsphere_centos
+    - name: vsphere_default
     - name: vsphere_flatcar
-    - name: gce_default
 
-- scenario: weave_containerd
-  initVersion: v1.28.3
+###################
+# CLUSTER UPGRADE #
+###################
+
+- scenario: upgrade_containerd
+  initVersion: v1.27.11
+  upgradedVersion: v1.28.7
   infrastructures:
-    - name: azure_default
-    - name: azure_centos
-    - name: azure_flatcar
-    - name: azure_rhel
-    - name: azure_rockylinux
-    - name: gce_default
+    - name: aws_centos_stable
+    - name: aws_default_stable
+    - name: aws_flatcar_stable
+    - name: aws_rhel_stable
+    - name: aws_rockylinux_stable
```

Comment:
Please replace AWS with Azure or GCP because AWS doesn't support in-tree CCM in v1.27.